### PR TITLE
Removes raw=true from download example links and updates gh-pages docs

### PR DESCRIPTION
**Pull request by nikhiljindal**
- Adding a munger to remove ?raw=true from download example links
- Processing all docs again using
  `go run _tools/release_docs/main.go _tools/release_docs/api-reference-process.go --branch release-1.1 --output-dir _v1.1 --version v1.1`

cc @bgrant0607

### CODE REVIEW

File: _v1.1/docs/admin/cluster-large.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To avoid running into cloud provider quota issues, when creating a cluster with
 
 ### Addon Resources
 
-To prevent memory leaks or other resource issues in [cluster addons](https://releases.k8s.io/v1.1.0/cluster/addons) from consuming all the resources available on a node, Kubernetes sets resource limits on addon containers to limit the CPU and Memory resources they can consume (See PR [#10653](http://pr.k8s.io/10653/files) and [#10778](http://pr.k8s.io/10778/files)).
+To prevent memory leaks or other resource issues in [cluster addons](https://releases.k8s.io/HEAD/cluster/addons) from consuming all the resources available on a node, Kubernetes sets resource limits on addon containers to limit the CPU and Memory resources they can consume (See PR [#10653](http://pr.k8s.io/10653/files) and [#10778](http://pr.k8s.io/10778/files)).
```
Comment (bgrant0607) [inline _v1.1/docs/admin/cluster-large.md]: This unversionized this link?

File: _v1.1/docs/admin/limitrange/index.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ---
 layout: docwithnav
-title: "</strong>"
+title: "title: \"Limit Range\""
```
Comment (bgrant0607) [inline _v1.1/docs/admin/limitrange/index.md]: Looks like a bug

File: _v1.1/docs/admin/cluster-large.md (hunk 1 above)
Comment (nikhiljindal) [inline _v1.1/docs/admin/cluster-large.md]: Looks like the script unversions all files that are not included in k8s.io: https://github.com/kubernetes/kubernetes/blob/gh-pages/_tools/release_docs/main.go#L66

File: _v1.1/docs/admin/limitrange/index.md (hunk 2 above)
Comment (nikhiljindal) [inline _v1.1/docs/admin/limitrange/index.md]: Yes. It happened in another doc

File: _v1.1/docs/admin/limitrange/index.md [hunk 3]
```diff
@@ -191,12 +195,14 @@ that runs the container unless the administrator deploys the kubelet with the fo
 
 ```
 {% raw %}
+{% raw %}
```
Comment (nikhiljindal) [inline _v1.1/docs/admin/limitrange/index.md]: This also looks like a bug.
It should not add raw, endraw again

File: _v1.1/docs/getting-started-guides/logging.md [hunk 4]
```diff
@@ -189,7 +189,7 @@ spec:
 {% endraw %}
 {% endhighlight %}
 
-[Download example](https://releases.k8s.io/v1.1.0/cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml?raw=true)
+[Download example](../../examples/blog-logging/counter-pod.yaml)
```
Comment (nikhiljindal) [inline _v1.1/docs/getting-started-guides/logging.md]: oops! This doesnt look right :)

File: _v1.1/docs/api.md [hunk 5]
```diff
@@ -83,12 +83,11 @@ In the future we expect that there will be more API groups, all at REST path `/a
...
 create their own API groups](design/extending-api.md), and to avoid naming collisions.
 
-## Enabling resources in the extensions group
+## Enabling the extensions group
 
-Jobs, Ingress and HorizontalPodAutoscalers are enabled by default.
-Other extensions resources can be enabled by setting runtime-config on
-apiserver. runtime-config accepts comma separated values. For ex: to enable deployments and disable jobs, set
-`--runtime-config=extensions/v1beta1/deployments=true,extensions/v1beta1/jobs=false`
+Enable `extensions/v1beta1` objects by adding the following flags to your API server:
```
Comment (bgrant0607) [inline _v1.1/docs/api.md]: This was your PR! #15981

File: _v1.1/docs/design/clustering/index.md [hunk 6]
```diff
@@ -1,6 +1,10 @@
 ---
 layout: docwithnav
-title: "</strong>"
+title: "title: \"Building with Docker\""
```
Comment (bgrant0607) [inline _v1.1/docs/design/clustering/index.md]: Another mis-munge.

File: _v1.1/docs/api.md (hunk 5 above)
Comment (nikhiljindal) [inline _v1.1/docs/api.md]: sent https://github.com/kubernetes/kubernetes/pull/16483

**Comment by nikhiljindal** [comment]
Ran the mungers once again (with https://github.com/kubernetes/kubernetes/pull/16484) and pushed a new version.

There seem to be 2 main issues:
* title is incorrect (layout and title are being added twice in many cases)
* raw, endraw tags are being added even when they are not required.

File: _v1.1/docs/getting-started-guides/logging.md (hunk 4 above)
Comment (nikhiljindal) [inline _v1.1/docs/getting-started-guides/logging.md]: https://github.com/kubernetes/kubernetes/pull/16484 will fix this

**Comment by nikhiljindal** [comment]
Superceeded by https://github.com/kubernetes/kubernetes/pull/16476

**Pull request closed without merging**